docs/reference/protege-faq.md

```diff
@@ -0,0 +1,7 @@
+# Protegé FAQs
+
+## How to escape characters in the class expression editor
+
+To add a GO term that contains `'` in its name (e.g. `RNA-directed 5'-3' RNA polymerase activity`) in the class expression editor, you need to escape the `'` characters. In Protegé 5.5 this is not automatically handled when you auto-complete with tab. To escape the character append `\` before the `'` -> `RNA-directed 5\'-3\' RNA polymerase activity`. You won't be able to add the annotation otherwise.
```

Comment:
```suggestion
To add an ontology term (such as a GO term) that contains `'` in its name (e.g. `RNA-directed 5'-3' RNA polymerase activity`) in the class expression editor, you need to escape the `'` characters. In Protegé 5.5 this is not automatically handled when you auto-complete with tab. To escape the character append `\` before the `'` -> `RNA-directed 5\'-3\' RNA polymerase activity`. You won't be able to add the annotation otherwise.
```
made this a bit more general

Reply:
@manulera 👀

Reply:
👍